src/unix/linux.c

```diff
@@ -2281,6 +2281,131 @@ uint64_t uv_get_available_memory(void) {
 }
 
 
+#define BUF_SIZE 1024
+
+static uv__cpu_constraint uv__get_cgroupv2_constrained_cpu(const char *cgroup) {
+  uv__cpu_constraint constraint = {0, 0, 0};
+  char path[256];
+  char buf[BUF_SIZE];
+  unsigned int weight;
+  int cgroup_size;
+  char *cgroup_trimmed;
+
+  // Trim ending \n by replacing it with a 0
+  cgroup_trimmed = cgroup + strlen("0::/");          // Skip the prefix "0::/"
```

Comment:
It's not wrong and with most compilers it amounts to the same generated machine code but idiomatic libuv source code uses sizeof for string literals:
```suggestion
  cgroup_trimmed = cgroup + sizeof("0::/") - 1;          // Skip the prefix "0::/"
```